integration_tests/test_domestic_migration.py

```diff
@@ -12,6 +12,29 @@ def test_individuals_move(simulants_on_adjacent_timesteps):
         ).all()
 
 
+def test_individuals_move_into_new_households(simulants_on_adjacent_timesteps):
+    for before, after in simulants_on_adjacent_timesteps:
+        new_household_movers = (
+            (before["household_id"] != after["household_id"])
+            & (after["housing_type"] == "Standard")
+            & (after["relation_to_household_head"] == "Reference person")
```

Comment:
Instead of saying they're a reference person after the move, why not say the household they moved into didn't exist in the before time-step? That seems more like the definition of a new household move to me. Then test that they're a reference person later.

Reply:
I was on the fence about that when writing this. I'll switch it!